driverVersions.json

```diff
@@ -82,7 +82,7 @@
         },
 	"74": {
 	    "driverNPMPackageVersion": "74.0.0",
-            "driverVersion": "74.0.3729.6"
+            "driverVersion": "74.0"
```

Comment:
if the next version goes on to be 75 everywhere, we should make the script more intelligent to use the version everywhere, so we don't need to keep updating

Reply:
sure, we'll see when version 75 is released